src/main/java/net/glowstone/chunk/GlowChunk.java

```diff
@@ -3,13 +3,8 @@
 import io.netty.buffer.ByteBuf;
 import io.netty.buffer.Unpooled;
 import it.unimi.dsi.fastutil.longs.Long2ObjectOpenHashMap;
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Set;
+
+import java.util.*;
```

Comment:
Simply adding an import would work, and would be better.

Reply:
Will fix it. IntelliJs automatic import optimize was on :/